charts/java-spring-boot/templates/deployment.yaml

```
@@ -0,0 +1,131 @@
+            - name: config
+              mountPath: {{ .Values.imageWorkDir | default "/deployments" }}/config
+          {{- if .Values.javaTruststore.enabled }}
+            - name: java-truststore
+              mountPath: {{ default "/etc/pki/ca-trust/extracted/java" .Values.javaTruststore.mountPath }}
+              readOnly: true
+          {{- end }}
+          {{- $credentials := .Values.credentials }}
+          {{- range $name, $credential := $credentials }}
+            - name: secret-{{ $name }}
+              mountPath: {{ $.Values.imageWorkDir | default "/deployments" }}/config/{{ $name }}
+              readOnly: true 
```

YAMLlint (1.35.1)

[error] trailing spaces

(trailing-spaces)